src/plugins/intel_gpu/src/graph/impls/ocl/fully_connected.cpp

```diff
@@ -273,6 +273,8 @@ attach_fully_connected_impl::attach_fully_connected_impl() {
         std::make_tuple(data_types::f32, format::bs_fs_fsv8_bsv8),
         std::make_tuple(data_types::f16, format::bs_fs_fsv8_bsv8),
         std::make_tuple(data_types::f16, format::bs_fs_fsv8_bsv16),
+        std::make_tuple(data_types::i8, format::bfzyx),
+        std::make_tuple(data_types::u8, format::bfzyx),
```

Comment:
which kernel does it use? I think we should use `fully_connected_imad`, but it should be fixed as well..

Reply:
Shared explanation about how fully_connected_imad was selected for the target by this fix.

Reply:
Could you explain the detail within the commit message / and here? I think it would be difficult to understand without explanation.

Reply:
Added